client/cluster-client/src/main/kotlin/uk/ac/ebi/biostd/client/cluster/api/LocalClusterClient.kt

```diff
@@ -34,6 +37,10 @@ class LocalClusterClient : ClusterClient {
         return if (process.isAlive) "RUNNING" else "DONE"
     }
 
+    override suspend fun jobLogs(jobId: String): String {
+        return jobLogs.getValue(jobId.toLong()).readText()
```

Comment:
I'm just curious, why do you use `getValue` instead of `jobLogs[jobId.toLong()] `?

Reply:
nullable vs not nullable